controller/model_provider/manager.py

```diff
@@ -0,0 +1,26 @@
+import os
+from typing import Any, List
+from graphql_api.types import ModelProviderInfoResult
+
+from util import service_requests
+
+BASE_URI = os.getenv("MODEL_PROVIDER")
+
+
+def get_model_provider_info() -> List[ModelProviderInfoResult]:
+    url = f"{BASE_URI}/info"
+    return service_requests.get_call_or_raise(url)
+
+
+def model_provider_download_model(
+    project_id: str, model_name: str, revision: str
```

Comment:
why do we need a project_id and revision to download a model?

Reply:
I added it for the websocket notifications, is there a better way to use websockets without the project id?

Reply:
The revision is not need, will remove it, but the project_id is needed for the notifications

Reply:
Regarding the revision, at the beginning I implemented the model-provider service so that it is also capable of managing multiple revisions of the same model, as this is not the plan anymore, we could remove this functionality in the service itself and the gateway

Reply:
project_id in notifications is somewhat problematic since models are globally available and with this only the specific project would be notified. If person A downloads in project A a model and person B somewhat simultaneously tries to download the same model in project B this will result in some issues. 

I'd go without the project_id and a global subscription in the frontend. However, this would only solve the issue for multiple projects within the same org. For multiple orgs multiple notifications need to be sent (one for each org). If there are questions regarding the how please ask :)

Reply:
@FelixKirsch please do 🙂

Reply:
changed in gateway, some changes in the model-provider are still needed